mpf/config_spec.yaml

```diff
@@ -1292,17 +1289,21 @@ segment_displays:
     __valid_in__: machine
     __type__: device
     number: single|str|
-    platform: single|str|None
     platform_settings: single|dict|None
+    platform: single|str|None
+light_segment_displays_device:
+    lights: list|dict(str:machine(lights))|
+    type: single|enum(7segment,bcd,14segment,16segment)|
 segment_display_player:
     __valid_in__: machine, mode, show
     __type__: config_player
     priority: single|int_or_token|0
     text: single|str|None
-    color: list|color|ffffff
+    color: list|color|None
     action: single|enum(add,remove,flash,no_flash,flash_match,set_color)|add
     key: single|str|None
     expire: single|ms_or_token|None
+    platform_options: single|dict|None
```

Comment:
This is probably still a leftover, right?

Reply:
Correct. Another one I missed.